### DISCUSSION TITLE
Support for old Targus Bluetooth mouse

### DISCUSSION THREAD

**Discussion by DavidAMcL** [noah-nuebling/mac-mouse-fix#448] [Questions]
I have a Targus Soft-Touch Bluetooth Mouse (AMB09US).  Mac-mouse-fix sort of recognizes it -- for example, the "Invert direction" works.  But I can't get it to do anything with the middle mouse button.  (It has only 3 buttons.)  Scrolling and everything else seem to work.

Is there anything I can do to enable the use of the middle button?  Thanks.

**Comment by noah-nuebling** [comment]
Hey @DavidAMcL, usually if certain buttons don't work with MMF, that's because your mouse sends special signals when you click that button which are meant to be picked up by the mouse's official driver software. 

But from a quick Google Search I doesn't seem like your mouse comes with a driver by Targus, not even on Windows, so this is weird. 

Maybe the button on your mouse is broken? You can use this website to see if your button works: https://www.onlinemictest.com/mouse-test/

Let me know what you find!
